core/cfdefs/cfdef_standard.php

```diff
@@ -448,11 +448,13 @@ function cfdef_input_radio( array $p_field_def, $p_custom_field_value, $p_requir
  * @return void
  */
 function cfdef_input_textbox( array $p_field_def, $p_custom_field_value, $p_required = '' ) {
-	echo '<input ' . helper_get_tab_index() . ' type="text" id="custom_field_' . $p_field_def['id'] . '" name="custom_field_' . $p_field_def['id'] . '" size="80"' . $p_required;
+	echo '<input ' . helper_get_tab_index() . ' type="text" id="custom_field_' . $p_field_def['id'] . '" name="custom_field_' . $p_field_def['id'] . '" ' . $p_required;
 	if( 0 < $p_field_def['length_max'] ) {
 		echo ' maxlength="' . $p_field_def['length_max'] . '"';
+		echo ' size="' .  $p_field_def['length_max'] . '"';
```

Comment:
Shouldn't this be min( max_length, 80 )?

Reply:
thanks i overlooked that